docs/getting_started.rst

```diff
@@ -14,7 +16,38 @@ Things You Need
 Installation Quick Reference Guide
 ==================================
 
-Placeholder
+.. Do we have a troubleshooting piece here incase something doesn't work?
+
+.. rst-class:: compact
+
+1. Get the your board NengoFPGA ready using the
+   :ref:`device-specific board setup documentation <board-setup>`.
+#. :ref:`Install NengoFPGA <software-install>`.
+#. :ref:`Edit the NengoFPGA config file <nengofpga-config>` to match your setup.
+#. Test NengoFPGA by running the ID extractor script:
+
+   i. In a terminal on your computer navigate to the root `nengo-fpga` directory.
+   #. Run the ID extractor script with::
+
+         python nengo_fpga/id_extractor.py <board>
+
+      where **<board>** is the board name as it appears in `fpga_config`.
+      See the `Copy Protection`_ section for more information.
+
+#. Now with the Device ID available, consult the
+   :ref:`device-specific bitstream documentation <bitstreams>` for how to obtain
+   usable bitstreams.
```

Comment:
I can't comment on the actual file itself (because it doesn't show up in the diff), but we should add an example output (use an id of `0x012345679ABCDEFGH` or something) to the copy protection section.

Something like:
> Running the `id_extractor.py` script will produce an output like
> ```
> Found ID 0x012345679ABCDEFGH (I can't remember the exact output)
> Saving to 'blah'
> ```

Also should add instructions on how to send us the ID (attach the file or send the number directly to our support email)